src/main/java/TaskManager.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Stores and add to the list of tasks.
+ */
+public class TaskManager {
+    private static final int MAX_TASK_SIZE = 100;
+    private static final String BY_LITERAL = "/by";
+    private static final String AT_LITERAL = "/at";
+    private static final int MAX_INPUT_PARAMS = 2;
+
+    private final Task[] tasks;
+    private int taskCount;
+
+    public TaskManager() {
+        tasks = new Task[MAX_TASK_SIZE];
+        taskCount = 0;
+    }
+
+    /**
+     * Adds a task to the user's list of tasks.
+     * Prints to user that task has been added.
+     *
+     * @param taskType represents the type of task to store.
+     * @param inputText text of the user input to store.
+     */
+    public void addTask(TaskType taskType, String inputText) {
+        Task task;
+        String[] descAndTime;
+
+        switch (taskType) {
+        case TODO:
+            task = new Todo(inputText);
+            break;
+        case DEADLINE:
+            descAndTime = splitItem(TaskType.DEADLINE, inputText);
+            task = new Deadline(descAndTime);
+            break;
+        case EVENT:
+            descAndTime = splitItem(TaskType.EVENT, inputText);
+            task = new Event(descAndTime);
+            break;
+        default:
+            throw new IllegalStateException("Unexpected value: " + taskType);
```

Comment:
I like the way you implement Java error checking by throw exception.

Reply:
Thanks!